third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -285,7 +285,7 @@ bool TargetInfo::warpReduce(RewriterBase &rewriter, Location loc,
                             unsigned interleave) const {
   auto b = TritonLLVMOpBuilder(loc, rewriter);
 
-  if (getISAFamily() == ISAFamily::CDNA4 &&
+  if (isCDNA(getISAFamily()) && getISAFamily() == ISAFamily::CDNA4 &&
```

Comment:
This is updated with https://github.com/triton-lang/triton/commit/23b8a7d6091ecbf224bcb7b7ce0d6d02a7394604. Can you merge in lastest `main`?